### ISSUE TITLE
Better explanation of "skipped"

### ISSUE THREAD

**Issue by edgeeffect** [OxfordEcosystemsLab/gemdata#14]
As an outsider coming in, I was very confused by the ForestPlots importer skipping rows without any explanation of why they were skipped. There error reporting could be a lot clearer.

**Comment by ashenkin** [comment]
Yep.  Perhaps good to remember that, while the traits importers will have a rather limited usage, the forestplots importer should have a much longer lifespan and higher usage.  So perhaps worthwhile to implement better error reporting at some point.
